prov/sockets/src/sock_fabric.c

```diff
@@ -265,6 +265,7 @@ int sock_verify_info(uint32_t version, struct fi_info *hints)
 	case FI_SOCKADDR_IN:
 		break;
 	default:
+		SOCK_LOG_DBG("Unsupported address format\n");
```

Comment:
@shefty, there is no macro SOCK_LOG_INFO, so SOCK_LOG_DBG is used everywhere in sockets provider.
May be you want to create such macro and replace each DBG with INFO?